### PR TITLE
Include LICENSE in compiled jar

### PR DESCRIPTION
**Pull request by pahimar**
Also added a licenses block to the pom

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,6 +8,13 @@
     <artifactId>api</artifactId>
     <version>4.0.0-SNAPSHOT</version>
 
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://opensource.org/licenses/mit-license.php</url>
```
Comment (connor4312) [inline pom.xml:14]: Need copyright holder / date?

File: src/main/resources/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Beam Interactive, Inc.
```
Comment (connor4312) [inline src/main/resources/LICENSE]: Should read `Copyright (c) Microsoft`

**Review by connor4312 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (pahimar) [reply pom.xml:14]: Examples I've seen have shown that for the licenses block, they point to the central definition of the license. The included license in the repo is considered the master license for the project. We do not need the licenses stanza to be compliant, however - it's just extra.

Do we have a central posted license, outside of our repos, that we can link to?

File: src/main/resources/LICENSE (hunk 2 above)
Comment (pahimar) [reply src/main/resources/LICENSE]: Fixed

File: pom.xml (hunk 1 above)
Comment (connor4312) [reply pom.xml:14]: Let me PM CELA

**Review by connor4312 — APPROVED** [review]

**Pull request merged**